config/config.go

```diff
@@ -32,6 +32,7 @@ const (
 	IpfsEndPoint                   = "ipfs_end_point"
 	IpfsTimeout                    = "ipfs_timeout"
 	LogKey                         = "log"
+	MaxMessageSize                 = "max_message_size"
```

Comment:
This is currently in bytes, lets make it in MB or GB and handle conversion. Also we need to enforce a lower and upper limit.